firewall/interception/windowskext/kext.go

```diff
@@ -142,9 +91,14 @@ func Stop() error {
 	}
 	ready.UnSet()
 
-	rc, _, lastErr := kext.stop.Call()
-	if rc != windows.NO_ERROR {
-		return formatErr(lastErr, rc)
+	err := closeDriver(kextHandle)
```

Comment:
With how we handle this now, maybe it does not make sense to check for `!ready.IsSet()` and just check if `kextHandle` is valid and should be closed instead.